supervision/detection/tools/inference_slicer.py

```diff
@@ -193,10 +218,13 @@ def _generate_offset(
                 of the image to be sliced.
             slice_wh (Tuple[int, int]): A tuple representing the desired width and
                 height of each slice.
-            overlap_ratio_wh (Tuple[float, float]): A tuple representing the desired
-                overlap ratio for width and height between consecutive slices. Each
-                value should be in the range [0, 1), where 0 means no overlap and a
-                value close to 1 means high overlap.
+            overlap_ratio_wh (Optional[Tuple[float, float]]): A tuple representing the
+                desired overlap ratio for width and height between consecutive slices.
+                Each value should be in the range [0, 1), where 0 means no overlap and
+                a value close to 1 means high overlap.
+            overlap_wh (Optional[Tuple[int, int]]): A tuple representing the desired
+                overlap for width and height between consecutive slices. Each value
```

Comment:
Can we say it's in pixels? I'd be slightly confused about measurement units.

Reply:
💯